backend/seeds/seeds.go

```diff
@@ -305,9 +305,9 @@ func setupAdminUser(store store.Store, username, password string) error {
 	}
 
 	admin := &types.User{
-		Username: username,
-		Password: hash,
-		Groups:   []string{"cluster-admins"},
+		Username:     username,
+		PasswordHash: hash,
```

Comment:
I think we should continue to specify the hashed password in the `Password` field here too.